frontend/src/app/document/content-field-preview/content-field-preview.component.html

```diff
@@ -0,0 +1,17 @@
+<ng-container *ngIf="field && document">
+    <ng-container *ngIf="document.highlight && document.highlight[field.name];
+        else unhighlightedRow">
+        <ng-container *ngFor="let highlight of document.highlight[field.name];
+            last as isLast">
```

Comment:
Lots of `ngIf` and `ngFor` here. I wonder if an elvis operator could cut some lines?

Reply:
E.g. something like `*ngFor=let highlight of document?.highlight[field?.name]`
Probably wishful thinking :)